tree/30_generic_methods/file_key_value_present_option.cf

```diff
@@ -36,10 +36,11 @@ bundle agent file_key_value_present_option(file, key, value, separator, option)
 {
   vars:
       "old_class_prefix" string => canonify("file_key_value_present_${file}");
-      "promisers"          slist => { @{this.callers_promisers}, cf_null }, policy => "ifdefined";
-      "class_prefix"      string => canonify(join("_", "promisers"));
       "args"              slist => { "${file}", "${key}", "${value}","${separator}", "${option}" };
 
+      "report_param"      string => join("_", args);
+      "class_prefix"      string => canonify("${old_class_prefix}_${report_param}");
```

Comment:
This may easily break if this string gets longer than 1024 chars

Reply:
I think it fails clearly though (and could could check this in the technique editor)